src/intervals/rounding.jl

```diff
@@ -187,7 +248,9 @@ function make_interval(T, expr1, expr2)
 end
 
 
-float(x::Interval) = make_interval(Float64, x)
+float(x::Interval) = Interval(convert(Float64,x.lo),convert(Float64,x.hi))
```

Comment:
If you round (further) then `float(@interval(0.1))` will not be equal to `@interval(0.1)`, as it is now, since it will be double rounded. This is tricky but important, since the trig functions (and #63) use it a lot! (Maybe, this was the reason I was not able to implement ^ using bigfloats with 53 bits, but I really forgot this.)